tests/test_evaluate_predictions.py

```diff
@@ -0,0 +1,34 @@
+from REL.evaluate_predictions import compare_and_count_entities, print_scores
+
+
+def test_perfect():
+    gold_entities = [ [ "1", "1" ] ]
+    predicted_entities = [ [ "1", "1" ] ]
+    correct, wrong_md, wrong_el, missed = compare_and_count_entities(gold_entities, predicted_entities)
+    precision_md, recall_md, f1_md, precision_el, recall_el, f1_el = print_scores(correct, wrong_md, wrong_el, missed)
+    assert [precision_md, recall_md, f1_md, precision_el, recall_el, f1_el] == [100, 100, 100, 100, 100, 100], "should be perfect MD and perfect EL"
```

Comment:
Suggestion to make it a bit easier to see what's going on here:

```suggestion
    counts = compare_and_count_entities(gold_entities, predicted_entities)
    scores = print_scores(*counts)
    assert scores == [100, 100, 100, 100, 100, 100], "should be perfect MD and perfect EL"
```